docs/spark-procedures.md

```diff
@@ -493,6 +493,35 @@ CALL spark_catalog.system.add_files(
 )
 ```
 
+### `register_table`
+
+Creates a catalog entry for a metadata.json file which already exists but does not have a corresponding catalog identifier.
```

Comment:
Should we add a warning about registering tables which exist in another catalog to current catalog? Because in that case table exists in both catalogs.

Reply:
Yeah, I have added a warning.